mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -491,11 +515,23 @@ func expandNodeConfig(v interface{}) *container.NodeConfig {
 			if data["count"].(int) == 0 {
 				continue
 			}
-			guestAccelerators = append(guestAccelerators, &container.AcceleratorConfig{
+			guestAcceleratorConfig := &container.AcceleratorConfig{
 				AcceleratorCount: int64(data["count"].(int)),
 				AcceleratorType:  data["type"].(string),
 				GpuPartitionSize: data["gpu_partition_size"].(string),
-			})
+			}
+
+			if v, ok := data["gpu_sharing_config"]; ok {
+				if len(v.([]interface{})) > 0 {
```

Comment:
these two if statements can be condensed
```golang
if v, ok := data["gpu_sharing_config"]; ok && len(v.([]interface{})) > 0 {
```

Reply:
done